projects/wonka-ui/karma.conf.js

```diff
@@ -11,7 +11,6 @@ module.exports = function (config) {
       require('karma-jasmine-html-reporter'),
       require('karma-coverage-istanbul-reporter'),
       require('@angular-devkit/build-angular/plugins/karma'),
-      require('karma-spec-reporter')
```

Comment:
I added karma-spec-reporter in order to have a cleaner report in the CI. It gives a list of all the tests and indicates which ones pass or not. For this to work, we need 'progress' as reporters.

Reply:
Ah ok. Fixed.